addons/udes_stock/controllers/stock_picking_batch.py

```diff
@@ -173,7 +173,8 @@ def create_batch_for_user(self,
     def update_batch(self, ident,
                      continue_batch,
                      move_line_ids,
-                     location_barcode=None):
+                     location_barcode=None,
+                     result_package=None):
```

Comment:
This needs to go in the specs (README).

Reply:
Maybe we can revisit the naming.
We're using `result_package_name` for `pick` and `result_package_id` for move line. This is extremely confusing.

Reply:
As agreed with @richardwhitefoot, will use `result_package_name`.